src/shared/util.js

```diff
@@ -144,6 +144,28 @@ const AnnotationBorderStyleType = {
   UNDERLINE: 5,
 };
 
+const AnnotationActionEventType = {
+  E: "MouseEnter",
+  X: "MouseExit",
+  D: "MouseDown",
+  U: "MouseUp",
+  Fo: "Focus",
+  Bl: "Blur",
+  PO: "PageOpen",
```

Comment:
`PageOpened` would be more consistent with `PageClosed` below. Is there a spec for these event names? In https://www.adobe.com/content/dam/acom/en/devnet/acrobat/pdfs/js_api_reference.pdf#page=16 I only find _some_ of them.

Reply:
I changed `PageClosed` into `PageClose`:
https://www.adobe.com/content/dam/acom/en/devnet/acrobat/pdfs/js_api_reference.pdf#page=387